.github/workflows/release.yaml

```diff
@@ -21,15 +25,17 @@ jobs:
     runs-on: [self-hosted, macos, arm64, 11, release]
     timeout-minutes: 60
     steps:
-      - uses: actions/setup-go@v4
-        with:
-          go-version: 1.19.x
       - uses: actions/checkout@v3
         with:
           fetch-depth: 1
           submodules: recursive
           persist-credentials: false
 
+      - uses: actions/setup-go@v4
+        with:
+          go-version: 1.19.x
+          cache: ${{ env.GOCACHE }}
```

Comment:
`GOCACHE` Go environment variable is normally a path to the cache, not bool. I'm in favor of setting this to `false` directly everywhere it's used

Reply:
Sure, added and pushed.